legend-pure-code-compiled-core-relational/src/main/resources/core_relational/relational/executionPlan/tests/executionPlanTest.pure

```diff
@@ -1,1699 +1,1714 @@
+      '  connection = TestDatabaseConnection(type = "H2")\n' +
+      ')\n';
+
+  let plan = executionPlan($query, $mapping, $runtime, meta::pure::router::extension::defaultRelationalExtensions());
+  assertEquals($expected, $plan->planToString(meta::pure::router::extension::defaultRelationalExtensions()));
+}
+
+function <<test.Test>> meta::pure::executionPlan::tests::testSupportStreamFlagSimple():Boolean[1]
+{
+   let plan = executionPlan({id:Integer[1], name:String[*]|Person.all()->filter(p|$p.name->in($name) || $p.age == $id)->project(col(p|$p.firstName, 'firstName'))}, simpleRelationalMapping, meta::relational::tests::testRuntime(), meta::pure::router::extension::defaultRelationalExtensions());
+   let functionParameters = $plan.rootExecutionNode->allNodes(defaultRelationalExtensions())->filter(node | $node->instanceOf(FunctionParametersValidationNode))->cast(@FunctionParametersValidationNode).functionParameters;
+
+   assertEquals($functionParameters->filter(param | $param.name == 'id').supportsStream, false);
+   assertEquals($functionParameters->filter(param | $param.name == 'name').supportsStream, true);
+}
+
+function <<test.Test>> meta::pure::executionPlan::tests::testSupportStreamFlagWithSupportedAndUnSupportedUsages():Boolean[1]
+{
+   let plan = executionPlan({name:String[*]|Person.all()->filter(p|$p.name->in($name) || $name->isEmpty())->project(col(p|$p.firstName, 'firstName'))}, simpleRelationalMapping, meta::relational::tests::testRuntime(), meta::pure::router::extension::defaultRelationalExtensions());
+   let functionParameters = $plan.rootExecutionNode->allNodes(defaultRelationalExtensions())->filter(node | $node->instanceOf(FunctionParametersValidationNode))->cast(@FunctionParametersValidationNode).functionParameters;
+
+   assertEquals($functionParameters->filter(param | $param.name == 'name').supportsStream, false);
+}
+
+function <<test.Test>> meta::pure::executionPlan::tests::testSupportStreamFlagWithTdsJoinForOneDB():Boolean[1]
+{
+   let plan =executionPlan({name:String[*]|
+                testJoinTDS_Person.all()->filter(p|$p.firstName->in($name))->meta::pure::tds::project([col(p|$p.firstName, 'firstName'), col(p|$p.employerID, 'eID')])
+                ->join(testJoinTDS_Firm.all()->filter(p|$p.legalName->in($name))->project([col(p|$p.firmID, 'fID'), col(p|$p.legalName, 'legalName')]), JoinType.INNER, {a,b|$a.getInteger('eID') == $b.getInteger('fID');});},testJoinTDSMapping, meta::relational::tests::testRuntime(), meta::pure::router::extension::defaultRelationalExtensions());
+   let functionParameters = $plan.rootExecutionNode->allNodes(defaultRelationalExtensions())->filter(node | $node->instanceOf(FunctionParametersValidationNode))->cast(@FunctionParametersValidationNode).functionParameters;
+
+   assertEquals($functionParameters->filter(param | $param.name == 'name').supportsStream, true)
```

Comment:
This is such a big diff :(

Reply:
there might be a spaces/newline related issue. Please configure your diff viewer to ignore them